stores/emoji_store.jsx

```diff
@@ -173,6 +174,28 @@ class EmojiStore extends EventEmitter {
         return recentEmojis;
     }
 
+    saveEmojiItems(emojiItems) {
+        const loadedEmojis = this.getEmojiItems();
+        if (loadedEmojis.length < emojiItems.length) {
+            localStorage.setItem(LOADED_EMOJI_ITEMS, JSON.stringify(emojiItems));
+        }
+    }
+
+    getEmojiItems() {
+        let emojiItems;
+        try {
+            emojiItems = JSON.parse(localStorage.getItem(LOADED_EMOJI_ITEMS));
+        } catch (e) {
+            // Error is handled below
+        }
+
+        if (!emojiItems) {
+            return [];
+        }
+
+        return emojiItems;
+    }
```

Comment:
No action needed now, but this should be moved to use Redux once the browser store is moved over